src/commands/tokens/create.ts

```diff
@@ -0,0 +1,49 @@
+// token create command
+
+import { Command, CommandArgs, CommandResult, help, success, failure, ErrorCodes, shortName, longName, hasArg, required } from "../../util/commandline";
+import { out } from "../../util/interaction";
+import { reportToken } from "./lib/format-token";
+import { MobileCenterClient, models, clientRequest } from "../../util/apis";
+
+const debug = require("debug")("mobile-center-cli:commands:apps:create");
+import { inspect } from "util";
+
+@help("Create a new API token")
+export default class TokenCreateCommand extends Command {
+  constructor(args: CommandArgs) {
+    super(args);
+  }
+
+  @help("Description of the app")
```

Comment:
Shouldn't this be "description of the token"? You're not dealing with an app here.